{"1.0.0":{"info":{"author":"Patrick","author_email":"patrsc@nubu.at","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Programming Language :: Python :: 3","Programming Language :: Python :: 3.10","Programming Language :: Python :: 3.11","Programming Language :: Python :: 3.9"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/patrsc/fastapi-simple-errors","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"fastapi-simple-errors","package_url":"https://pypi.org/project/fastapi-simple-errors/","platform":null,"project_url":"https://pypi.org/project/fastapi-simple-errors/","project_urls":{"Bug Tracker":"https://github.com/patrsc/fastapi-simple-errors/issues","Documentation":"https://github.com/patrsc/fastapi-simple-errors","Homepage":"https://github.com/patrsc/fastapi-simple-errors","Repository":"https://github.com/patrsc/fastapi-simple-errors"},"provides_extra":null,"release_url":"https://pypi.org/project/fastapi-simple-errors/1.0.0/","requires_dist":["fastapi<0.116.0,>=0.115.0"],"requires_python":"<4.0.0,>=3.8.1","summary":"Simple error handling for fastapi using custom error classes.","version":"1.0.0","yanked":false,"yanked_reason":null},"last_serial":25157909,"urls":[{"comment_text":"","digests":{"blake2b_256":"14c692a3d6590ecf6c44267cd1a3dc9367942336ccec8521502988441da54278","md5":"3e5ea13a41af5e16a204d463393e6404","sha256":"23371d9c21fa31897254845ab32196f1ff229df5a790748ff23c40b2692e8516"},"downloads":-1,"filename":"fastapi_simple_errors-1.0.0-py3-none-any.whl","has_sig":false,"md5_digest":"3e5ea13a41af5e16a204d463393e6404","packagetype":"bdist_wheel","python_version":"py3","requires_python":"<4.0.0,>=3.8.1","size":6255,"upload_time":"2024-09-23T17:25:30","upload_time_iso_8601":"2024-09-23T17:25:30.035126Z","url":"https://files.pythonhosted.org/packages/14/c6/92a3d6590ecf6c44267cd1a3dc9367942336ccec8521502988441da54278/fastapi_simple_errors-1.0.0-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"ab9abeff67f1d2004edea041078ce3836632740d592d6b055794627b078ef845","md5":"9963ceec1af38363a8379f4d1886d475","sha256":"c14f6117d2fb68c62775ae0af060228f439e20148f4aec608f4cceaee0a5f8ee"},"downloads":-1,"filename":"fastapi_simple_errors-1.0.0.tar.gz","has_sig":false,"md5_digest":"9963ceec1af38363a8379f4d1886d475","packagetype":"sdist","python_version":"source","requires_python":"<4.0.0,>=3.8.1","size":5765,"upload_time":"2024-09-23T17:25:31","upload_time_iso_8601":"2024-09-23T17:25:31.644857Z","url":"https://files.pythonhosted.org/packages/ab/9a/beff67f1d2004edea041078ce3836632740d592d6b055794627b078ef845/fastapi_simple_errors-1.0.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}